app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Send me reset password instructions" %>
```

Comment:
- [ ] When I try to request a new password I get this error. You can fix this by disabling `turbo` like so:

  ```
  <%= f.submit "Send me reset password instructions", data: { turbo: false } %>
  ```

  ![djdk](https://user-images.githubusercontent.com/36542986/195471046-055c0b5c-dfb0-4bb1-a8f7-ce63e7b8884e.jpg)

Reply:
@TedLivist Awesome, good catch there, thank you very much for it.